infra/network-isolation.bicep

```diff
@@ -0,0 +1,143 @@
+param vnetName string
+param location string = resourceGroup().location
+param tags object = {}
+
+param appServicePlanId string
+param appServicePlanName string
+param storageAccountId string
+param searchServiceId string
+param openAiId string
+param searchServiceName string
+param resourceToken string
+
+var environmentData = environment()
+
+module vnet './core/networking/vnet.bicep' = {
+  name: vnetName
+  params: {
+    name: vnetName
+    location: location
+    tags: tags
+    appServicePlanId: appServicePlanId
+    appServicePlanName: appServicePlanName
+  }
+}
+
+
+// DNSs Zones
```

Comment:
Do these all follow some convention for dns zone name? I don't understand how they work, since they don't reference a resource nor do they get output. A comment/link might help?

Reply:
They're used by the PE later in the file, the PE assigns a record from the Zone.